x-pack/plugins/lists/server/services/exception_lists/exception_list_client.ts

```diff
@@ -295,6 +309,7 @@ export class ExceptionListClient {
     type,
   }: UpdateExceptionListItemOptions): Promise<ExceptionListItemSchema | null> => {
     const { savedObjectsClient, user } = this;
+    await this.createTrustedAppsList();
```

Comment:
I don't think we need to create the list before attempting an update or find.

Reply:
Yeah, I think I'd expect to just get a 404 here if the list doesn't exist.

Reply:
FYI - I was attempting to avoid the `404` on "find". I really would like this "list" to just be assumed to always be there. That being said, I will address that in the new API